doc/contrib/documentation.md

```diff
@@ -0,0 +1,84 @@
+# Documentation
+
+This document gives guidelines for contributors concerning Elektra's documentation.
+This document takes preference to the actual situation, so if you see documentation
+not according to this document, please [create an issue](https://issue.libelektra.org)
+or directly fix it with your next PR.
+
+> Note: It is always allowed to improve the documentation, in every PR, even if
+> the documentation fix is completely unrelated.
+
+## Target Groups
+
+We write documentation for three groups:
+
+- User:
+  Is a person who uses some tool of Elektra, e.g. `kdb`, Qt-Gui or Web-UI.
+- Developer:
+  Is a person who uses some library of Elektra within an application, tool or plugin.
+- Contributor
+  Is a person who does changes within Elektra's repository.
+
+There are 3 separate folders for these 3 groups:
+
+- [for users](/doc/help)
+- [for developers](/doc/dev)
+- [for contributors](/doc/contrib)
+
+> Takeaway:
+> Every document must have a clear target group (user, developer or contributor).
+> Sometimes it is clear from the directory sometimes it must be explicitly stated,
+> e.g. tutorials.
```

Comment:
Why not create separate folders for tutorials too?

Beyond that a tutorial should probably state it's target audience as a sub-group of the 3 main groups, e.g. "Developers who want to write a backend plugin".

Reply:
For tutorials we already have a quite nice structure (even finer than 3 target groups):
https://www.libelektra.org/tutorials/readme
I wouldn't change what already works.